languages/en.json

```diff
@@ -17,5 +17,11 @@
   "MYTHACRI.MayhemTitle": "Mayhem: {name}",
   "MYTHACRI.MayhemCurrentPoints": "You currently have {points} Mayhem Points.",
   "MYTHACRI.MayhemConsume": "Consume Mayhem Points",
-  "MYTHACRI.MayhemCannotDeduct": "You do not have enough Mayhem Points available!"
+  "MYTHACRI.MayhemCannotDeduct": "You do not have enough Mayhem Points available!",
+  "MYTHACRI.WeaponPropertyParry": "Parrying",
+  "MYTHACRI.WeaponPropertyScatter": "Scatter",
+  "MYTHACRI.WeaponPropertySuperheavy": "Superheavy",
+  "MYTHACRI.WeaponPropertyRocket": "Rocket",
+  "MYTHACRI.WeaponPropertyHeat": "Heat",
+  "MYTHACRI.WeaponPropertyTwinshot": "Twinshot",
```

Comment:
```diff
- "MYTHACRI.WeaponPropertyTwinshot": "Twinshot",
+ "MYTHACRI.WeaponPropertyTwinshot": "Twinshot"
```
A trailing comma in a json will cause the file to not load.

Reply:
Okay NOW it should be fixed